### ISSUE TITLE
JSDoc comments break `indent`

### ISSUE THREAD

**Issue by j-f1** [eslint/typescript-eslint-parser#344]
**What version of TypeScript are you using?**
2.3.4

**What version of `typescript-eslint-parser` are you using?**
4.0.0

**What code were you trying to parse?**

```ts
/**
 * @a
 */
foo;
```

This code also breaks:

```ts
/**
 * @a
 */
/**
 * a
 */
foo;
```

**What did you expect to happen?**
No errors

**What happened?**
> Expected indentation of 0 spaces but found 1. (indent)

**None of these throw the error**:
```ts
/**
 * a
 */
foo;
```

```ts
/**
 * @a
 */
```

```ts
/**
 * @a
 */
```

**Comment by w0rp** [comment]
I've been seeing this issue myself in my codebase. In some cases, we were able to disable the errors by using `/***` for comments, so they won't be recognised as jsdoc comments, or by wrapping the jsdoc comments in eslint comments to disable the rule for those lines.

**Comment by JamesHenry** [comment]
I haven't had chance to dig into this at all yet, but it's possible you can use the new functionality in the `indent` rule to work around this: https://github.com/eslint/eslint/issues/8594

**Comment by bradzacher** [comment]
what's the status of this? it's producing a lot of erroneous linting errors across my codebase.

**Comment by sarvaje** [comment]
@bradzacher if it is an option for you, you can use the rule [`indent-legacy`](https://eslint.org/docs/rules/indent-legacy) until this issue is fixed

**Comment by JamesHenry** [comment]
Thanks for providing the alternative workaround, @sarvaje.

I am going to try and come up with a solution for this "integration" issue.

**Comment by JamesHenry** [comment]
I can no longer reproduce this issue with the latest dependencies involved. Please feel free to reopen if it is still an issue.

**Comment by Tarnadas** [comment]
It's still not resolved for me.

These are my dependencies:
´´´json
    "typescript": "^2.6.2",
    "typescript-eslint-parser": "^12.0.0",
    "eslint": "^4.12.1"
```

**Comment by bkotos** [comment]
This is not resolved for me either.  Having the latest `typescript-eslint-parser` doesn't fix it.
My relevant dependencies:
```
    "typescript": "^2.6.1",
    "typescript-eslint-parser": "^12.0.0",
    "eslint": "^4.14.0",
```

**Comment by stanbluijs** [comment]
Having the same issue, is there any update on this?
```
"typescript": "^2.6.2",
"typescript-eslint-parser": "^12.0.0",
"eslint": "^4.16.0"
```

**Comment by benedictchen** [comment]
+1
